Prototope/TextLayer.swift

```diff
@@ -56,7 +95,31 @@ public class TextLayer: Layer {
 			updateSizePreservingOrigin() // Adjust width/height as necessary for new wrapping mode.
 		}
 	}
-
+	
+	public var textAlignment: Alignment {
+		get {
+			return Alignment(nsTextAlignment: label.textAlignment)
+		}
+		set {
+			label.textAlignment = newValue.toNSTextAlignment()
+			//No need to adjust size, since changing alignment doesn't influence it
+		}
+	}
+	
+	/** Distance from top of layer to the first line's baseline */
+	public var baselineHeight: Double {
+		return Double(label.font.ascender)
+	}
+	
+	/** Aligns this layer with the first baseline of the other layer */
+	public func alignWithBaselineOf(otherLayer: TextLayer) {
```

Comment:
Seems weird to have a one-off layout method for this Layer. Not knowing what the use case of baseline will be, I'd say we can leave this out for now. What do you think?